### DISCUSSION TITLE
Why so many MQTT subscription topics are on `result` same subscription topics?

### DISCUSSION THREAD

**Discussion by hemangjoshi37a** [arendst/Tasmota#17701] [Q&A] [answered]
For so many devices and sensors there is same MQTT subscription topic that is `results` and because of that we are having issue developing a Flutter over it. Can we have topic for each seonsor on different name please ?

**Comment by arendst** [comment, marked as answer]
SetOption4 1

**Reply by hemangjoshi37a** [reply]
I think this should be a default. Because setting a common subscirbe topic then a specific is better to parse the received data as well as to remember what was the subscribing topic. If you allow me I can open Pull Request to make this change.

**Reply by arendst** [reply]
As always, legacy. No need for a PR.
